src/components/signup-card/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Card, Input, Button, Col, Row } from 'antd';
+import './style.less';
+
+const SignupCard = (props) => {
+    return (
+      <Card className="card-signup">
+        <p style={{fontSize: 22, marginBottom: 10}}>Sign Up</p>
```

Comment:
Why use inline styles here and not less?

Reply:
removed all inlines c: